documentation/src/main/resources/pages/ditto/protocol/examples/things/generated/commands/modify/modifyfeature.md

```diff
@@ -11,6 +11,12 @@
       "y": 2.718,
       "z": 1,
       "unit": "g"
+    },
+    "desiredProperties": {
```

Comment:
I don't see the file `modifyfeaturersponsecreated.md` also being changed in this PR - but it should be as the created JSON content is returned when the feature gets created.

Reply:
The file is created and should be displayed as before.. 
[documentation/src/main/resources/pages/ditto/protocol/examples/things/generated/commands/modify/modifyfeatureresponsecreated.md](url)

Reply:
Yes, you're right